portable/GCC/ARM_CRx_No_GIC/portASM.S

```diff
@@ -193,7 +329,7 @@ FreeRTOS_IRQ_Handler:
     ISB
 
     /* Write to the EOI register. */
-    LDR     r0, ulICCEOIRConst
+    LDR     r0, =ulICCEOIR
     LDR     r2, [r0]
     STR     r0, [r2]
```

Comment:
I know that this is code that is already here, but this makes an assumption that the MCU being used has a memory-mapped EOI register, the [ARM_CRx_MPU:FreeRTOS_IRQ_Handler](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/V11.1.0/portable/GCC/ARM_CRx_MPU/portASM.S#L439-L451) removed this code, I'd recommend doing the same here.

Reply:
The suggestion is good but this may break existing applications that rely on this behavior. I'd keep it here for backward compatibility.